### ISSUE TITLE
Groundwater module error

### ISSUE THREAD

**Issue by rclavera** [landlab/landlab#1118]
Dear Landlab team,

I was testing the groundwater component. The example used have the next magnitudes:
number of rows:        2761
number of columns:  2528
number of nodes:      6979808
number of cells:         6969234
number of core cells: 6969234
area of a cell:       1.0

This error appears running the example:
calc_recharge_flux_in
    return np.sum(self._grid.area_of_cell * self._recharge[self._cores])
ValueError: operands could not be broadcast together with shapes (6969234,) (4946676,)

I understand that 
“self._grid.area_of_cell” is an array of 6969234 (cells)
“self._recharge” is an array of 6979808 (nodes)
“self._recharge[self._cores]” is an array of 4946676 (core nodes)

What can I do?
Thank you very mcuh

**Comment by kbarnhart** [comment]
@rclavera thank you for making this issue. With the information you have provided it is really difficult to assess the issue. 

Could you please provide a code-snippet that contains all of the code necessary to reproduce your error (e.g., small model grid, all synthetic values, no input files). This will make it possible to reproduce, diagnose, assess the error you've found. 

tagging @DavidLitwin

**Comment by rclavera** [comment]
@kbarnhart ,
Here I attached the code
[test.py.zip](https://github.com/landlab/landlab/files/4089573/test.py.zip)
sorry, but the other file (compressed) occupies 35.7 MB. What can I do?
Thank you

**Comment by rclavera** [comment]
@kbarnhart thanks, 
I'm working on it reducing the image .
Is not Landlab v2 available in PYPI?

**Comment by kbarnhart** [comment]
@rclavera Landlab v2 is on pypi but you need to specify the `--pre` flag. 

```
pip install --upgrade --pre landlab
```

should work to upgrade. Let me know if it doesn't work for you. Note that we presently don't have python 3.8 builds on either conda-forge or pypi. we should have them in the next few days. 

I've now created a PR (#1119) to fix the [update instructions](https://landlab.readthedocs.io/en/master/install/index.html#id2)

**Comment by rclavera** [comment]
@kbarnhart, Landlab v2 has been installed. Thanks

**Comment by rclavera** [comment]
I attach the files to test the problem I have.
[Archive 2.zip](https://github.com/landlab/landlab/files/4101473/Archive.2.zip)

**Comment by kbarnhart** [comment]
@DavidLitwin could you please have a look at this?

**Comment by DavidLitwin** [comment]
@kbarnhart  and @rclavera I will have a look tomorrow.

**Comment by DavidLitwin** [comment]
@rclavera I just submitted PR #1124 to fix the issue. It corrects both flux in and storage calculations. Also, I made some changes to your test code so that it will run on Landlab 2.0. It's attached here. 

Unless there are any further issues related to those the flux calculations, I will close this when the PR is complete. Thanks for catching the bug!

[test.zip](https://github.com/landlab/landlab/files/4109091/test.zip)
